modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/unit/UnitDeployCommand.java

```diff
@@ -72,6 +73,7 @@ public Integer call() throws Exception {
                 .output(spec.commandLine().getOut())
                 .errOutput(spec.commandLine().getErr())
                 .verbose(verbose)
+                .exceptionHandler(new ClusterNotInitializedExceptionHandler("Cannot deploy unit", "cluster init"))
```

Comment:
We should rewrite the UnitCommand to use `askQuestionIfNotConnected` and change this handler accordingly for non-REPL commands. Since there are a lot of them we might want to extract this to a pair of construction methods? Also since we renamed the CLI executable to `ignite3` we might want to do that now and then change the string accordingly.

Reply:
Probably yes, deserves a separate ticket for that.